perfact/zodbsync/commands/layer_update.py

```diff
@@ -74,6 +74,12 @@ def run(self):
             return
 
         self._playback_paths(sorted(paths))
+        self.sync.record(sorted(paths), recurse=False)
```

Comment:
This does not work in `dry-run` mode (the rollback already happened and not all paths are found) and even without it, I get a lot of errors that paths can't be recorded because they don't exist aymore (with `skip_errors=True`, it still works, but the output is not pretty).

Reply:
I think this whole plan does not work well with `dry-run`. For now, I will only do the re-record and conflict check in non-dry-run mode. Otherwise we would need to do the re-record before the rollback and also roll back the changes done by this on the file system. Which will be difficult if there are multiple phases...